tools/c7n_org/setup.py

```diff
@@ -48,9 +48,9 @@
     'long_description_content_type': 'text/markdown',
     'author': 'Cloud Custodian Project',
-    'author_email': None,
-    'maintainer': None,
-    'maintainer_email': None,
+    'author_email': 'None',
```

Comment:
Seems a bit weird to stringify the `None` here especially for the optional maintainer parameters. That's an upstream concern though, and not a blocker. Just looked odd at a glance.